packages/preview/dmi-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "dmi-thesis"
+version = "0.1.0"
+entrypoint = "src/main.typ"
+authors = ["Nico Bachmann <typst@nifalu.ch>"]
+license = "MIT"
+description = "A Typst template for DMI theses"
```

Comment:
The description usually shouldn't contain typst, since it's a bit redundant. See: https://github.com/typst/packages/blob/main/docs/manifest.md#writing-a-good-description